R/cieplot.R

```diff
@@ -19,10 +19,9 @@
 #'   from the plotting area when `space = "cielab"`.
 #' @param box logical. Should the plot area box and axes be plotted? (defaults
 #'   to `FALSE`)
-#' @param margin vector of four numbers specifying drawing margins for CIELAB
-#'   plot (defaults to c(0,0,0,0))
 #' @param ciebg should the colour background be plotted for CIEXYZ plot?
 #'   (defaults to `TRUE`)
+#' @param margin Deprecated. Please use the standard par() method for custom margins. 
```

Comment:
```suggestion
#' @param margin Deprecated. Please use the standard [par()] method for custom margins. 
```

for formatting and auto-linking